styles/components/_button-custom.scss

```diff
@@ -19,16 +18,11 @@
 
     &-secondary {
         border: solid 1px var(--color-text-secondary);
-    }
-
-    &-secondary-banner {
-        margin-top: 20px;
         justify-content: center;
         display: flex;
         flex-wrap: nowrap;
         padding: 20px 36px;
```

Comment:
if you set padding in here, login button at header will take this value and false compared to design